### ISSUE TITLE
[Keyboard navigation - Make code Arcade- Home Page]: Keyboard focus indicator is not visible on the 'Experiments' and 'Ok' buttons.

### ISSUE THREAD

**Issue by vroopea** [microsoft/pxt-microbit#6096]
### User Experience:
If the focus indicator is not visible on any control, then users with motor disabilities who depend on the keyboard for navigation will not be able to identify the focus on that control which can prevent them from performing actions that are important.

**Note:**
User credentials should NOT be included in the bug.

### Repro Steps:

1. Launch application URL: https://arcade.makecode.com/.
2. Home page will open.
3. Navigate to 'Version' button which is present at the bottom of the page and select it.
4. About dialog appears, navigate the dialog and observe whether keyboard focus indicator is visible on the 'Experiments' and 'Ok' buttons or not.

### Actual Result:
**Issue:** Keyboard focus indicator is not visible on the 'Experiments' and 'Ok' buttons.

### Expected Result:
Contrasting rectangular keyboard focus indicator should be visible on the 'Experiments' and 'Ok' buttons.

### MAS Reference:
[MAS 2.4.7 – Focus Visible](https://aka.ms/MAS2.4.7)

### ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:
NA

### UIA/DOM details available:
NA

### Reference Links:

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to an external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### Test Environment:

- OS Version: Windows 11 24H2 (OS Build:26100.3037)
- Browser: Chrome Version 132.0.6834.196 (Official Build) (64-bit)
- Test Matrix: Chrome + NVDA
- URL: https://arcade.makecode.com/

### Attachment:

<img width="980" alt="Image" src="https://github.com/user-attachments/assets/723a7be1-e890-4f46-9ce6-7b68b632b82e" />

**Comment by vroopea** [comment]
#Accessibility; #A11yMAS; #A11ySev2; #MAS2.4.7; #AILimited; #HCL; #Benchmark; #Win11-Chrome; #BM-HCL-MakeCode-Feb2025

**Comment by srietkerk** [comment]
Hi @vroopea, can you verify this again. This looks to be fixed when I test it.

When tabbed on the Experiments button:

![Image](https://github.com/user-attachments/assets/9c4a4b99-ec01-4b2b-a14a-3ff5bf68f8de)

When tabbed on the Ok button:

![Image](https://github.com/user-attachments/assets/da73e3dd-28c9-41e0-80c9-3b8b0402f08e)

**Comment by srietkerk** [comment]
@vroopea, closing as fixed. Please let me know if you're still having problems with this.
